api-tester/build.gradle

```diff
@@ -1,11 +1,6 @@
 apply from: "$rootProject.projectDir/base-application.gradle"
 
 android {
-    sourceSets {
```

Comment:
this didn't appear to be doing anything... seems like a legacy gradle thing. i believe this all just sets the default directories anyways